Tests/LSPLoggingTests/SupportTests.swift

```diff
@@ -25,8 +25,8 @@ final class SupportTests: XCTestCase {
 
     func check(expected: [(String, LogLevel)], file: StaticString = #file, line: UInt = #line) {
       testLogger.flush()
-      XCTAssert(messages.count == expected.count, "\(messages) does not match expected \(expected)", file: file, line: line)
-      XCTAssert(zip(messages, expected).allSatisfy({ $0.0 == $0.1 }), "\(messages) does not match expected \(expected)", file: file, line: line)
+        XCTAssert(messages.count == expected.count, "\(messages) does not match expected \(expected)", file: (file), line: line)
```

Comment:
I think the correct change here is that `#file` should be changed to `#filePath` in `func check` above.

There are a number of other places in this patch that are passing `(file)` that should be unnecessary once the declaration has been changed to use `#filePath`.

Also, there seems to be some unexpected changes to indentation that should be removed.

Reply:
Thanks @benlangmuir - I'll run back through it and revisit all the spots I tried to convert and update them, and fix the indentation while I'm at it.